tests/cluster/tests/29-cluster-response-with-tls.tcl

```diff
@@ -0,0 +1,141 @@
+# This test must be executed last
```

Comment:
We have a bunch of cluster tests in the normal testing framework, the ones that runs as part of the CI, like https://github.com/redis/redis/blob/unstable/tests/unit/cluster/slot-ownership.tcl . That seems better than just terminating all the existing tests here.

Reply:
Oh, it does look more neat. I was stuck in the cluster testing framework.

Reply:
Yeah, I didn't like the clustering tests so I ported a bunch of stuff over. Let me know if you can port it, I can help too if need be.